db/migrate/20200923192959_create_cavc_remand.rb

```diff
@@ -0,0 +1,22 @@
+class CreateCavcRemand < Caseflow::Migration
+  def change
+    create_table :cavc_remands do |t|
+      t.bigint "appeal_id", null: false, comment: "Appeal that CAVC is remanding"
```

Comment:
Should be past tense?
```suggestion
      t.bigint "appeal_id", null: false, comment: "Appeal that CAVC is remanded"
```

Reply:
```suggestion
      t.bigint "appeal_id", null: false, comment: "Appeal that CAVC has remanded"
```